go-packages/meep-redis/db.go

```diff
@@ -41,11 +43,20 @@ type Connector struct {
 // NewConnector - Creates and initialize a Redis connector
 func NewConnector(addr string, table int) (rc *Connector, err error) {
 	rc = new(Connector)
-	err = rc.connectDB(addr, table)
+
+	// Connect to Redis DB
+	for retry := 0; !rc.connected && retry <= dbMaxRetryCount; retry++ {
+		err = rc.connectDB(addr, table)
+		if err != nil {
+			log.Warn("Failed to connect to DB. Retrying... Error: ", err)
+			continue
+		}
+	}
```

Comment:
should had a few ms sleep to prevent retrying right away.... if it just failed.. chances it will fail again right away.....

Reply:
Same comment as before regarding timeouts.